drivers/entropy/entropy_stm32.c

```diff
@@ -224,7 +224,7 @@ static void configure_rng(void)
 #endif /* health_test_config */
 
 	if (cur_nist_cfg != desired_nist_cfg || cur_htcr != desired_htcr) {
-		MODIFY_REG(rng->CR, cur_nist_cfg, (desired_nist_cfg | RNG_CR_CONDRST));
+		stm32_reg_modify_bits(&rng->CR, cur_nist_cfg, (desired_nist_cfg | RNG_CR_CONDRST));
```

Comment:
By the way, drop the useless parentheses?
```suggestion
		stm32_reg_modify_bits(&rng->CR, cur_nist_cfg, desired_nist_cfg | RNG_CR_CONDRST);